.github/workflows/ci.yml

```diff
@@ -21,10 +21,18 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
+
+    - name: Use Node.js version ${{ matrix.node-version }}
       uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node-version }}
-    - run: yarn install --frozen-lockfile --non-interactive
-    - run: yarn lint
-    - run: yarn test
+
+    - name: Install dependencies
+      run: yarn install --frozen-lockfile --non-interactive
+
+    - name: Run ESLint
```

Comment:
This runs several types of linting, not just ESLint.

Reply:
Good catch, fixed. Thanks for the review!